src/main/java/Updater/Updater.java

```diff
@@ -0,0 +1,43 @@
+package Updater;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.awt.Desktop;
+
+public class Updater {
+
+    public static void main(String[] args) {
```

Comment:
@kmr-rohit The oldPath and newFilePath will be passed as arguments to this main method via `System[] args`.
It will work similar to any java program taking a command-line arguments.

Reply:
Done.

Reply:
Okay :+1: